.gitignore

```diff
@@ -55,3 +55,7 @@ docs/_build/
 
 # PyBuilder
 target/
+
+# Misc
```

Comment:
For me as a developer, pylarion is used only with betelgeuse and I like to keep betelgeuse and pylarion in one folder for my easy maintenance.  I thought this would help other developers too.  Having said that, I am happy to move this to my general .gitignore if this is not going to help fellow developers.